spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/SecretsConfigProperties.java

```diff
@@ -190,6 +189,28 @@ public String getNamespace() {
 			return labels;
 		}
 
+		@Override
```

Comment:
we did the same change in config maps: let's make secrets unique by `name` and `namespace`.